From 638a1b59c23908be23e5d16dea8649060bf85fc2 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 4 Nov 2016 15:05:58 +0800
Subject: [PATCH 1272/5242] MLK-13420 ARM: dts: imx: Correct the i2c3 pin's
 voltage setting

commit  4e84eca768ab004f6bd6b0d71ca03789de07e394 from
https://source.codeaurora.org/external/imx/linux-imx.git

On i.MX6SLL EVK and ARM2 board, the pins for I2C3 should be set
to 1.8V voltage(set the LVE bit to 1) to decrease the current
leakage from these pins.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll-evk.dts         |    4 ++--
 arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sll-evk.dts b/arch/arm/boot/dts/imx6sll-evk.dts
index 4b4e339..cac2781 100644
--- a/arch/arm/boot/dts/imx6sll-evk.dts
+++ b/arch/arm/boot/dts/imx6sll-evk.dts
@@ -607,8 +607,8 @@
 
 		pinctrl_i2c3: i2c3grp {
 			fsl,pins = <
-				MX6SLL_PAD_AUD_RXFS__I2C3_SCL  0x4001b8b1
-				MX6SLL_PAD_AUD_RXC__I2C3_SDA   0x4001b8b1
+				MX6SLL_PAD_AUD_RXFS__I2C3_SCL  0x4041b8b1
+				MX6SLL_PAD_AUD_RXC__I2C3_SDA   0x4041b8b1
 			>;
 		};
 
diff --git a/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts b/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts
index 19bb8ca..dc5d745 100644
--- a/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6sll-lpddr3-arm2.dts
@@ -640,8 +640,8 @@
 
 		pinctrl_i2c3: i2c3grp {
 			fsl,pins = <
-				MX6SLL_PAD_EPDC_SDCE2__I2C3_SCL  0x4001b8b1
-				MX6SLL_PAD_EPDC_SDCE3__I2C3_SDA  0x4001b8b1
+				MX6SLL_PAD_EPDC_SDCE2__I2C3_SCL  0x4041b8b1
+				MX6SLL_PAD_EPDC_SDCE3__I2C3_SDA  0x4041b8b1
 			>;
 		};
 
-- 
1.7.9.5

